src/lib/consensus/proof_of_stake_fuzzer.ml

```diff
@@ -137,7 +137,7 @@ module Vrf_distribution = struct
     let rec find_potential_proposals acc_proposals window_depth slot =
       let slot_in_dist_range = slot < dist.term_slot in
       let window_expired =
-        window_depth >= default_window_size && List.length acc_proposals > 0
+        window_depth >= default_window_size && acc_proposals <> []
```

Comment:
Nit: `<>` is monomorphised to `int`

Since we're not currently building this (it needs some fixing up), I won't block on this.

Reply:
It's fixed nonetheless. Let's not add junk code to the codebase.